release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -126,6 +126,29 @@ cluster installation by sharing the node's pull secret credentials with the
 
 {product-title} {product-version} introduces the Vertical Pod Autoscaler Operator (VPA). The VPA reviews the historic and current CPU and memory resources for containers in Pods and can update the resource limits and requests based on the usage values it learns. You create individual custom resources (CR) to instruct the VPA to update all of the Pods associated with a workload object, such as a Deployment, Deployment Config, StatefulSet, Job, DaemonSet, ReplicaSet, or ReplicationController. The VPA helps you to understand the optimal CPU and memory usage for your Pods and can automatically maintain Pod resources through the Pod lifecycle.
 
+[id="ocp-4-5-cluster-logging"]
+=== Cluster logging
+
+[id="ocp-4-5-elasticsearch-6"]
+==== Elasticsearch version upgrade
+
+Cluster logging in {product-title} {product-version} now uses Elasticseach 6.8.1 as the default log store. 
+
+The new Elasticsearch version introduces a new Elasticsearch data model that improves overall performance and scalability. With the new data model, data is no longer indexed by type, infrastructure, application and project. Data is only indexed by type:
+
+* The application logs that were previously in the *project-* indicies in {product-title} 4.4 are now in a single *app-* index.
+* The infrastructure logs that were previously in the *.opertions-* indices are now in the *infra-* indices. 
+* The audit logs are stored in the *audit-* indices.
+
+Because of the new data model, you will need to update your Kibana index patterns and visualizations to match the new indices.
+
+Elasticserach 6.x also introduces an enhanced security feature, Open Distro for Elasticsearch. Open Distro for Elasticsearch provides a comprehensive set of advanced security features designed to keep your data secure.
```

Comment:
introduces an alternate security plugin

Reply:
@jcantrill Thank you for the plugin wording. Was looking for that. 
It's not truly _alternate_, right? That sounds like I have a choice.
_introduces a new security plugin_?